projects/libs/core-library-typescript/src/common-library/grpc-server.ts

```diff
@@ -16,17 +24,27 @@ protos.set("VirtualMachine", "virtual-machine.proto");
 protos.set("SidechainConnector", "sidechain-connector.proto");
 protos.set("Management", "management.proto");
 
+const DEFAULT_MANAGEMENT_PORT = 8081;
```

Comment:
check with kirill if this should be configurable (make sure to ask him if there is a good reason and its not a yagni)